daemon/src/main.rs

```diff
@@ -19,10 +19,14 @@ struct Cli {
     #[command(subcommand)]
     command: Commands,
     /// Path to the Unix domain socket to use for communication between daemon and editors.
-    #[arg(short, long, global = true, default_value = DEFAULT_SOCKET_PATH)]
+    #[arg(
+      short, long, global = true,
+      default_value = DEFAULT_SOCKET_PATH,
+      env = "ETHERSYNC_SOCKET",
+    )]
     socket_path: PathBuf,
     /// Enable verbose debug output.
-    #[arg(short, long, global = true, action)]
```

Comment:
This doesn't seem to change behavior, as can be seen by the output of `ethersync --help`. If the intent was to allow `--debug=true`, I think you would need something like `action(clap::ArgAction::Set), default_value = "false"`, but I'm having difficulty figuring out how to make that work.

If the intent was just to make a simple boolean flag `-d`/`--debug`, I think this `action` argument is redundant. Let me know if I'm mistaken!

Reply:
Good catch. The intent was the latter, I'm not sure why we added it in the first place if it does nothing. I think a possible alternative is to make it a count by repeating it, like [here](https://docs.rs/clap/latest/clap/_derive/_tutorial/chapter_0/index.html).